src/utils.js

```diff
@@ -138,3 +138,6 @@ export const updateAttrsDialog =
       dialog.showModal()
     })
   }
+
+export const findExtension = (editor, extension) =>
+  editor.extensionManager.extensions.find((e) => e.name === extension)
```

Comment:
Moving it here and exporting it exposes `findExtension` in the `django-prose-editor/editor` module; it's not required (as commented above) and I'm not sure we want this right now.